pandas/core/ops.py

```diff
@@ -397,6 +397,169 @@ def _get_op_name(op, special):
+>>> df2 = pd.DataFrame({'tool': ['python', 'r', 'julia'],
+...                     'score': [300, 200, 100]},
+...                     columns=['tool', 'score'])
+>>> df2
+     tool  score
+0  python    300
+1       r    200
+2   julia    100
+>>> df1.lt(df2)
+    tool  score
+0  False   True
+1  False  False
+2  False  False
+"""
+
+_le_example_FRAME = """
+>>> df1 = pd.DataFrame({'tool': ['python', 'r', 'julia'],
+...                     'score': [100, 200, 300]},
+...                     columns=['tool', 'score'])
+>>> df1
+     tool  score
+0  python    100
+1       r    200
+2   julia    300
+>>> df2 = pd.DataFrame({'tool': ['python', 'r', 'julia'],
+...                     'score': [300, 200, 100]},
+...                     columns=['tool', 'score'])
+>>> df2
+     tool  score
+0  python    300
+1       r    200
+2   julia    100
+>>> df1.le(df2)
+   tool  score
+0  True   True
+1  True   True
+2  True  False
+"""
+
+_gt_example_FRAME = """
+>>> df1 = pd.DataFrame({'tool': ['python', 'r', 'julia'],
+...                     'score': [100, 200, 300]},
+...                     columns=['tool', 'score'])
+>>> df1
+     tool  score
+0  python    100
+1       r    200
+2   julia    300
+>>> df2 = pd.DataFrame({'tool': ['python', 'r', 'julia'],
+...                     'score': [300, 200, 100]},
+...                     columns=['tool', 'score'])
+>>> df2
+     tool  score
+0  python    300
+1       r    200
+2   julia    100
+>>> df1.gt(df2)
+    tool  score
+0  False  False
+1  False  False
+2  False   True
+"""
+
+_ge_example_FRAME = """
+>>> df1 = pd.DataFrame({'tool': ['python', 'r', 'julia'],
+...                     'score': [100, 200, 300]},
+...                     columns=['tool', 'score'])
+>>> df1
+     tool  score
+0  python    100
+1       r    200
+2   julia    300
+>>> df2 = pd.DataFrame({'tool': ['python', 'r', 'julia'],
+...                     'score': [300, 200, 100]},
+...                     columns=['tool', 'score'])
+>>> df2
+     tool  score
+0  python    300
+1       r    200
+2   julia    100
+>>> df1.ge(df2)
+   tool  score
+0  True  False
+1  True   True
+2  True   True
+"""
+
+_comp_others = """
+DataFrame.eq : Return DataFrame of boolean values equal to the
+    elementwise rows or columns of one DataFrame to another
+DataFrame.ne : Return DataFrame of boolean values not equal to
+    the elementwise rows or columns of one DataFrame to another
+DataFrame.le : Return DataFrame of boolean values less than or
+    equal the elementwise rows or columns of one DataFrame
+    to another
+DataFrame.lt : Return DataFrame of boolean values strictly less
+    than the elementwise rows or columns of one DataFrame to
+    another
+DataFrame.ge : Return DataFrame of boolean values greater than or
+    equal to the elementwise rows or columns of one DataFrame to
+    another
+DataFrame.gt : Return DataFrame of boolean values strictly greater
+    than to the elementwise rows or columns of one DataFrame to
+    another
+"""
```

Comment:
I'm not sure if I'm missing something, but is this always the same for all methods? It could go directly to the doctring and not  in a separate variable if that's the case. Besides good docstrings, it's good if we keep the code as simple as possible. So we usually leave the same `See Also` for all methods in this case, even if the method being documented is self referencing.

Also, do you think we can find something less verbose for the descriptions. Something like `Compare DataFrames for equality elementwise` may be?

Reply:
No, what I meant is that if it's possible, it'd be good that each of these descriptions is shorter. Just making them more concise if you find the way.